src/containers/CartPage.less

```diff
@@ -200,6 +200,69 @@
             border-color: @mainColor;
           }
         }
+
+        .btn-cmd-checkout {
+          display: inline-block;
+          padding: 8px 14px;
+          margin-bottom: 0;
+          font-size: 14px;
+          text-align: center;
+          vertical-align: middle;
+          cursor: pointer;
+          border-radius: 4px;
+          white-space: nowrap;
+          user-select: none;
+          border-radius: 2px;
+          margin-bottom: 2px;
+          color: #ffffff;
+          background-color: @mainColor;
+
+          &:focus {
+            outline: thin dotted #333;
+            outline: 5px auto -webkit-focus-ring-color;
```

Comment:
You can't define same css prop twice. One will get overridden. Pick one :) Also avoid vendor specific css. You probably need only first line.